.github/workflows/push.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Klepsydra Technologies AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Projects check
+
+on:
+  push:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  projectsbuild:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: kpsr-sdk
+    name: Check ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/klepsydra-technologies/kpsrbase:1.0.0
+    steps:
+      - name: Clone sources
+        uses: actions/checkout@v4
+        with:
+          repository: klepsydra-technologies/${{ matrix.name }}
+          ref: main
+          fetch-depth: 0
+          submodules: recursive
+
+      - uses: actions/checkout@v4
```

Comment:
This make no sense. This job should clone specific projects that have kpsr-build as submodule, and checkout kpsr-build submodule to that specific version, and try to build if it's compatible.

Reply:
At least it builts kpsr-build; the kpe-build [version](https://github.com/klepsydra-technologies/kpe-build/blob/f05fe336fccb8fe82abc795adcf9702551ba20f1/.github/workflows/push.yml) does not even run `make`
At some point (question above) kpsr-sdk is downloaded. Do you recommend building it to check that kpsr-build is ok? Add kpsr-core too?

Reply:
CI updated to build kpsr-sdk with the current commit of kpsr-build, is that ok @tomasz3dk?

Reply:
It's not needed. Just remove this.

Reply:
and the final Build, done.